src/main/java/seedu/siasa/logic/commands/contact/EditContactCommand.java

```diff
@@ -92,12 +90,13 @@ public CommandResult execute(Model model) throws CommandException {
 
         // Update all the associated policies
         ArrayList<Pair<Policy, Policy>> policiesToBeUpdated = new ArrayList<>();
-        model.updateFilteredPolicyList(new PolicyIsOwnedByPredicate(contactToEdit));
         List<Policy> contactPolicies = model.getFilteredPolicyList();
```

Comment:
Would this lead to a bug if the policylist was filtered e.g to contactpolicy of another contact since now some policies are not in the list?